Patches/Simple Ogre Race/CE_Ogre_Race.xml

```diff
@@ -0,0 +1,138 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+	<!-- It's all Ogre now -->
+	<Operation Class="PatchOperationFindMod">
+					<mods>
+						<li>Simple Ogre Race</li>
+					</mods>
+		<match Class="PatchOperationSequence">
+			<operations>
+
+			<li Class="PatchOperationAddModExtension">
+				<xpath>/Defs/AlienRace.ThingDef_AlienRace[defName="FP_OgreRace"]</xpath>
+				<value>
+					<li Class="CombatExtended.RacePropertiesExtensionCE">
+						<bodyShape>Humanoid</bodyShape>
+					</li>
+				</value>
+			</li>
+			
+			<!-- Melee Tool & Basestats Defs -->
+			<li Class="PatchOperationReplace">
+				<xpath>/Defs/AlienRace.ThingDef_AlienRace[defName="FP_OgreRace"]/statBases/Mass</xpath>
+					<value>
+      						<Mass>80</Mass>
+						<MeleeDodgeChance>1</MeleeDodgeChance>
+						<MeleeCritChance>1.5</MeleeCritChance>
+						<MeleeParryChance>1</MeleeParryChance>
+          						<Suppressability>0.75</Suppressability>
+						<CarryWeight>55</CarryWeight>

```

Comment:
`CarryBulk` and `CarryWeight` are multiplied by 1.5 due to the ogre's bodySize, leaving the final values at 82.5kg and 45 bulk respectively. Weight should probably be decreased somewhat, and bulk increased.

Reply:
I'm aware, this is the intended value. They are meant to be significantly stronger than base humans.
Also, can they not wear backpacks and vests?